libs/btp/navigation/components/navigation-link/navigation-link.component.ts

```diff
@@ -47,7 +48,7 @@ export class NavigationLinkRefDirective {
     ],
     host: {
         class: 'fd-navigation__link',
-        role: 'link'
+        '[role]': 'quickCreate ? "button" : "link"'
```

Comment:
```suggestion
        '[attr.role]': 'quickCreate ? "button" : "link"'
```

should fix the build